pkg/ebpf/oom-kill-kern-user.h

```diff
@@ -8,7 +8,7 @@
 #endif
 
 struct oom_stats {
-  char cgroup_name[64];
+  char cgroup_name[65];
```

Comment:
```suggestion
  char cgroup_name[129];
```

Reply:
Yes. Maybe it’s more future proof if a future version of docker/containerd/cri-o/etc. decides to switch to longer IDs.